Warning in clipbrd.h about wrong ifdef

I compile with warnings as errors, so this is causing an error on the current VS2022 (17.7.2):

1>d:\devtools\wx\trunk\include\wx\msw\clipbrd.h(18,1): warning C4574: 'WXWIN_COMPATIBILITY_3_2' is defined to be '0': did you mean to use '#if WXWIN_COMPATIBILITY_3_2'?

--- a/include/wx/msw/clipbrd.h
+++ b/include/wx/msw/clipbrd.h
@@ -15,7 +15,7 @@
 
 // Deprecated wxMSW-only function superseded by wxClipboard, don't use them and
 // use that class instead.
-#ifdef WXWIN_COMPATIBILITY_3_2
+#if WXWIN_COMPATIBILITY_3_2
 
 wxDEPRECATED_MSG("Use wxClipboard") WXDLLIMPEXP_CORE bool wxOpenClipboard();
 wxDEPRECATED_MSG("Use wxClipboard") WXDLLIMPEXP_CORE bool wxIsClipboardOpened();

--- a/src/msw/clipbrd.cpp
+++ b/src/msw/clipbrd.cpp
@@ -63,7 +63,7 @@
 static int gs_htmlcfid = 0;
 static int gs_pngcfid = 0;
 
-#ifdef WXWIN_COMPATIBILITY_3_2
+#if WXWIN_COMPATIBILITY_3_2
 
 static bool gs_wxClipboardIsOpen = false;
 

